config/rbac/daemon_role.yaml

```diff
@@ -72,3 +72,21 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - apps.kruise.io
+  resources:
+  - nodepodprobes
+  verbs:
+  - get
+  - list
+  - patch
```

Comment:
why daemon need to update nodeprobe, it seems that updates for nodepodprobe/status is enough